templates/zerver/rate_limit_exceeded.html

```diff
@@ -0,0 +1,21 @@
+{% extends "zerver/portico.html" %}
+
+{% block portico_content %}
+
+<div class="error_page">
+    <div class="container">
+        <div class="row-fluid">
+            <img src="/static/images/500art.svg" alt=""/>
+            <div class="errorbox">
+                <div class="errorcontent">
+                    <h1 class="lead">{{ _("You are doing that too much.") }}</h1>
+                    <p>
+                        {% trans %}Try again in: {{retry_after}} seconds.{% endtrans %}
```

Comment:
Nit, but I'm unsure about the colon.  Maybe:
```suggestion
                        {% trans %}You can try again in {{retry_after}} seconds.{% endtrans %}
```